File: front/src/api/settle/sheet/model/settleSheetSummaryBo.ts
export interface SettleSheetSummaryBo {
  supplierId: string;
  supplierCode: string;
  supplierName: string;
  /**
   * 未对账单数量
   */
  unCheckSheetNum: number;
  /**
   * 未对账单总金额
   */
  unCheckTotalAmount: number;
  /**
   * 未结算单数量
   */
  unSettleSheetNum: number;
  /**
   * 未结算单总金额
   */
  unSettleTotalAmount: number;
  /**
   * 部份结算单数量
   */
  partSettleSheetNum: number;
  /**
   * 部份结算单总金额
   */
  partSettleTotalAmount: number;
  /**
   * 已结算单数量
   */
  settledSheetNum: number;
  /**
   * 已结算单总金额
   */
  settledTotalAmount: number;
}

File: backend/xingyun-settle/src/main/java/com/lframework/xingyun/settle/bo/sheet/SettleSheetSummaryBo.java
package com.lframework.xingyun.settle.bo.sheet;

import lombok.Data;

import java.io.Serializable;
import java.math.BigDecimal;

/**
 * 供应商结算汇总结果
 */
@Data
public class SettleSheetSummaryBo implements Serializable {

    private static final long serialVersionUID = 1L;

    private String supplierId;

    private String supplierCode;

    private String supplierName;
    /**
     * 未对账单数量
     */
    private Integer unCheckSheetNum = 0;
    /**
     * 未对账单总金额
     */
    private BigDecimal unCheckTotalAmount = BigDecimal.ZERO;
    /**
     * 未结算单数量
     */
    private Integer unSettleSheetNum = 0;
    /**
     * 未结算单总金额
     */
    private BigDecimal unSettleTotalAmount = BigDecimal.ZERO;
    /**
     * 部份结算单数量
     */
    private Integer partSettleSheetNum = 0;
    /**
     * 部份结算单总金额
     */
    private BigDecimal partSettleTotalAmount = BigDecimal.ZERO;
    /**
     * 已结算单数量
     */
    private Integer settledSheetNum = 0;
    /**
     * 已结算单总金额
     */
    private BigDecimal settledTotalAmount = BigDecimal.ZERO;
}
